internal/util/csiconfig.go

```diff
@@ -164,11 +176,21 @@ func GetClusterID(options map[string]string) (string, error) {
 	return clusterID, nil
 }
 
-func GetNetNamespaceFilePath(pathToConfig, clusterID string) (string, error) {
+func GetRBDNetNamespaceFilePath(pathToConfig, clusterID string) (string, error) {
+	cluster, err := readClusterInfo(pathToConfig, clusterID)
+	if err != nil {
+		return "", err
+	}
+
+	return cluster.RBD.NetNamespaceFilePath, nil
+}
+
+// GetCephFSNetNamespaceFilePath returns the netNamespaceFilePath for CephFS volumes.
+func GetCephFSNetNamespaceFilePath(pathToConfig, clusterID string) (string, error) {
 	cluster, err := readClusterInfo(pathToConfig, clusterID)
 	if err != nil {
 		return "", err
 	}
 
-	return cluster.NetNamespaceFilePath, nil
+	return cluster.CephFS.NetNamespaceFilePath, nil
```

Comment:
ditto

Reply:
https://github.com/ceph/ceph-csi/blob/devel/internal/cephfs/mounter/kernel.go#L59-L62 and https://github.com/ceph/ceph-csi/blob/devel/internal/cephfs/mounter/fuse.go#L73-L77